### ISSUE TITLE
On councillors' pages, the link to view all primary sponsored legislation shows all sponsored legislation

### ISSUE THREAD

**Issue by fgregg** [datamade/nyc-councilmatic#109]
http://nyc.councilmatic.org/search/?q=&selected_facets=sponsorships_exact:Margaret%20S.%20Chin 

Shows all the legislation sponsored by Margaret Chin, not just the legislation of which she is the primary sponsore.

**Comment by cathydeng** [comment]
yeah we are using all sponsors to facet, not primary sponsor. idk if the search facet is a bug, but it is kind of confusing to have a link next to a list of primary-sponsored bills on person page. maybe we can re-word the link
